lib/recurly/ApiError.js

```diff
@@ -1,7 +1,7 @@
 class ApiError extends Error {
   constructor (message, type, params) {
     super(message)
-    this.name = 'RecurlyApiError'
+    this.name = this.constructor.name
```

Comment:
How about `this.name = 'Recurly' + this.constructor.name` so it looks like `RecurlyValidationError`?